walkthroughs/week-1-web-development/examples/stanley/src/main/java/com/google/sps/ServerMain.java

```diff
@@ -17,7 +17,7 @@
   public static void main(String[] args) throws Exception {
 
     // Create a server that listens on port 8080.
-    Server server = new Server(8080);
+    Server server = new Server(2000);
```

Comment:
No need to change this, but to find the pid (process id) of a process listening on a port and kill it so you can use the port again, you can follow along at https://stackoverflow.com/a/18706913. I only know this works because I spent one semester running a bazillion processes in an OS class :)

Reply:
ah okay I forgot about killing the prior server, just used this as a work-around